tools/build_h3_tools.sh

```diff
@@ -186,8 +186,9 @@ LDFLAGS=${LDFLAGS:-"-Wl,-rpath,${OPENSSL_LIB}"}
 
 # Then nghttp3
 echo "Building nghttp3..."
-[ ! -d nghttp3 ] && git clone --depth 1 -b v1.1.0 https://github.com/ngtcp2/nghttp3.git
+[ ! -d nghttp3 ] && git clone --depth 1 -b v1.2.0 https://github.com/ngtcp2/nghttp3.git
 cd nghttp3
+git submodule update --init
```

Comment:
As per their [notes](https://github.com/ngtcp2/nghttp3/releases/tag/v1.2.0) we need to fetch some of the dependencies.

Reply:
Ah, great observation. I think I missed this in my previous update.